docs/pages/eas-update/debug.mdx

```diff
@@ -1,69 +1,87 @@
 
-## expo-updates configuration
+## Go to the Deployments page
 
-The expo-updates library runs inside an end-user's app and makes requests to an update server to get the latest update.
+The EAS website has a [Deployments page](https://expo.dev/accounts/[account]/projects/[project]/deployments) that shows the current state of our app. The term _deployment_ refers to a group of builds and their updates. If we've made builds and updates with EAS, you can see your project's status on the website in the Deployments tab.
 
-### Verifying app configuration
+<ImageSpotlight
+  alt="Deployments tab"
+  src="/static/images/eas-update/deployments-website-tab.png"
+  style={{ maxWidth: 400 }}
+/>
 
-When we set up EAS Update, we likely ran `eas update:configure` to configure expo-updates to work with EAS Update. This command makes changes to our app config (**app.json**/**app.config.js**). Here are the fields we'd expect to see:
+## Common problems
 
-- `runtimeVersion` should be set. By default, it is `{ "policy": "sdkVersion" }`. If our project has **android** and **ios** directories, we'll have to set the `runtimeVersion` manually.
-- `updates.url` should be a value like `https://u.expo.dev/your-project-id`, where `your-project-id` matches the ID of our project. We can see this ID on [our website](https://expo.dev/accounts/[account]/projects/[project]).
-- `updates.enabled` should not be `false`. It's `true` by default if it is not specified.
+The following section describes common problems and how to fix them. Below is a diagram of how EAS Update works and the spots that are useful to inspect when finding the root cause of an issue. In the following sections, we'll inspect and verify these spots and more.
 
-Finally, make sure that `expo-updates` is included in **package.json**. If it's not, run:
+<ImageSpotlight alt="Map of debugging spots" src="/static/images/eas-update/debug-map.png" />
 
-<Terminal cmd={['$ npx expo install expo-updates']} />
+### Unexpected channel
 
-### Inspecting expo-updates configuration after prebuild
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-channel.png"
+  style={{ maxWidth: 600 }}
+/>
 
-Whenever we run `eas build`, the `npx expo prebuild` command is run on our project on EAS' servers to unpack the **android** and **ios** directories that contain native files. This makes it so EAS Build can build any project, whether it includes the native files or not.
+If the deployment channel is unexpected, it means your build was not built with the correct channel. To fix this, [configure your channel](/#configure-channel).
```

Comment:
Emphasize that a rebuild of the app is necessary after changing this (common mistake).